### ISSUE TITLE
Contour plot does not respect line properties

### ISSUE THREAD

**Issue by PeterPablo** [matlab2tikz/matlab2tikz#592]
**Consider:**
```
[C,h] = contourf(peaks(5));
set(h,'LineColor','r');
set(h,'LineStyle',':');
set(h,'LineWidth',2);
```

**Output from R2014b:**
![image](https://cloud.githubusercontent.com/assets/4340267/6780714/f6a73878-d167-11e4-8838-bc7db060e26a.png)

I was unable to locate an issue that is describing this. Line properties are currently not respected. (May be @okomarov you want to deal with this, while having a look at #582?).

**Comment by PeterPablo** [comment]
Note: The color of one contour being green instead of blue might be due to how the path is closed...

**Comment by egeerardyn** [comment]
Oops, I fixed this by accident in #721. :smiley:

**Comment by PeterPablo** [comment]
:+1: :smile: (I had the feeling there was already an issue report for this...)
